src/Symfony/Component/Form/Tests/Extension/Validator/Constraints/FormValidatorTest.php

```diff
@@ -149,11 +127,11 @@ public function testValidateIfChildWithValidConstraint()
         $this->assertNoViolation();
     }
 
-    public function testDontValidateIfParentWithoutCascadeValidation()
+    public function testDontValidateIfParentWithoutValidConstraint()
```

Comment:
Let me phrase this another way: Is there any way to make these two tests fail (individually) without making at least one other test in this test class fail at the same time?